lib/pbkit/pbkit.h

```diff
@@ -75,17 +73,20 @@ void pb_wait_until_gr_not_busy(void);
 DWORD pb_wait_until_tiles_not_busy(void);
 
 DWORD   *pb_begin(void);    //start a block with this (avoid more than 128 dwords per block)
-void    pb_push1to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1); //slow functions but with debug messages
-void    pb_push2to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2);
-void    pb_push3to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3);
-void    pb_push4to(DWORD subchannel, DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3, DWORD param4);
-void    pb_push1(DWORD *p, DWORD command, DWORD param1); //slow functions but with debug messages (targets SUBCH_3D (0))
-void    pb_push2(DWORD *p, DWORD command, DWORD param1, DWORD param2);
-void    pb_push3(DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3);
-void    pb_push4(DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3, DWORD param4);
-void    pb_push4f(DWORD *p, DWORD command, float param1, float param2, float param3, float param4);
-void    pb_push_transposed_matrix(DWORD *p, DWORD command, float *m);
-void    pb_end(DWORD *pEnd);    //end a block with this (triggers the data sending to GPU)
+inline DWORD   *pb_push_to(DWORD subchannel, DWORD *p, DWORD command, DWORD nparam);
```

Comment:
Compiler is likely to just ignore this `inline` and iirc we are not using lto. An alternative approach could be to define it as a static function in this header file so the compiler can optimize in the object file

Reply:
Described in PR description. Consider this preparation for #156 

Moving code into the header would be an ugly temporary hack and could cause issues with object files of different pbkit versions.

Reply:
> Moving code into the header would be an ugly temporary hack

Not really? Just put them in a .h or .inl and `#include` it

> could cause issues with object files of different pbkit versions
Huh? This is not an issue. We aren't keeping old object files around, and Make resolves the include dependency